rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
Please add the rental itself to the @person to achieve 'belongs-to' association.

Reply:
Done.